lts.js

```diff
@@ -0,0 +1,79 @@
+const fs = require("fs/promises");
+const path = require("path");
+
+(async () => {
+  const LTS_SCHEDULE_URL =
+    "https://raw.githubusercontent.com/nodejs/Release/main/schedule.json";
+
+  const versions = await fetch(LTS_SCHEDULE_URL).then((r) => r.json());
+
+  const currentLTS = +Object.entries(versions)
+    .map(([version, metadata]) => ({
+      version: version.replace(/\D/g, ""),
+      lts: metadata.lts ?? false,
```

Comment:
I try to stay far away from checking if things are `NaN` because Javascript isn't good at it. What you have here looks like it should be right, but I've been bitten by `NaN` too many times. 😬

Reply:
fwiw, Javascript is a *lot* better than it used to be if you use `Number.isNan` but i understand :D